### PR TITLE
[Minor] refactor test code

### PR DESCRIPTION
**Pull request by advancedxy**
### What changes were proposed in this pull request?
Refactor some test code

### Why are the changes needed?
For better code quality. And fixes some flaky test such as #388 

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Existing UTs. There should be no logic change.

### CODE REVIEW

File: server/src/test/java/org/apache/uniffle/server/StorageCheckerTest.java [hunk 1]
```diff
@@ -48,16 +49,19 @@ public static void clear() {
   }
 
   @Test
-  public void checkTest() throws Exception {
+  public void checkTest(@TempDir File baseDir) throws Exception {
     ShuffleServerConf conf = new ShuffleServerConf();
     conf.setBoolean(ShuffleServerConf.HEALTH_CHECK_ENABLE, true);
     conf.setString(ShuffleServerConf.RSS_STORAGE_TYPE, StorageType.LOCALFILE.name());
-    conf.set(ShuffleServerConf.RSS_STORAGE_BASE_PATH, Arrays.asList("st1", "st2", "st3"));
```
[the remark this answers is not in the document]
Comment (zuston) [reply server/src/test/java/org/apache/uniffle/server/StorageCheckerTest.java:55]: You are right. Could u help do some cleanup after test finished?

**Review by zuston — APPROVED** [review]
LGTM. 

I will approve first. If you have some changes, please amend them

File: server/src/test/java/org/apache/uniffle/server/StorageCheckerTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (zuston) [reply server/src/test/java/org/apache/uniffle/server/StorageCheckerTest.java:55]: You are right.

**Review by jerqi — APPROVED** [review]
LGTM

**Review by zuston — APPROVED** [review]
LGTM

**Pull request merged**